db/schema_tables.cc

```diff
@@ -1690,6 +1717,16 @@ std::vector<mutation> make_create_keyspace_mutations(lw_shared_ptr<keyspace_meta
     map["class"] = keyspace->strategy_name();
     store_map(m, ckey, "replication", timestamp, map);
 
+    if (features.contains<schema_feature::SCYLLA_KEYSPACES>()) {
+        schema_ptr scylla_keyspaces_s = scylla_keyspaces();
+        mutation scylla_m(scylla_keyspaces_s, pkey); // pkey can be reused, it's identical in both tables
+        auto storage_map = keyspace->get_storage_options().to_map();
+        if (!storage_map.empty()) {
+            store_map(scylla_m, ckey, "storage", timestamp, storage_map);
+        }
+        mutations.emplace_back(std::move(scylla_m));
```

Comment:
I asked above to fail create ks statement if unrecognized option is found, but at least we're ignoring them here